### PR TITLE
Admin: Show licensed roles and unit membership in UI

### PR DESCRIPTION
**Pull request by Clarity-89**
**What this PR does / why we need it**:
Display user roles and org membership in the list view (if available). 

**Which issue(s) this PR fixes**:

Part of https://github.com/grafana/grafana-enterprise/issues/1869
Fixes: https://github.com/grafana/grafana-enterprise/issues/2005

**Special notes for your reviewer**:
The UI still might have some minor changes but we're hoping to get this into tomorrow's beta release so we can iterate on it a bit more before stable is out.

Screenshot 2021-09-29 at 16 43 27

### CODE REVIEW

File: public/app/features/admin/UserListAdminPage.tsx [hunk 1]
```diff
@@ -149,6 +179,16 @@ const renderUser = (user: UserDTO) => {
           </a>
         )}
       </td>
+      {showLicensedRole && (
+        <td className={cx('link-td', styles.iconRow)}>
+          <a className="ellipsis" href={editUrl} title={user.name} aria-label={`Edit user's ${user.name} details`}>
```
Comment (hugohaggmark) [inline public/app/features/admin/UserListAdminPage.tsx:184]: Hmm, thinking about all these repetitive aria-labels, can we find a way to reduce them?

**Review by hugohaggmark — COMMENTED** [review]
Code looks great as always 🎉  Haven't been able to test it yet and I left a comment.

**Comment by Clarity-89** [comment]
> Code looks great as always 🎉 Haven't been able to test it yet and I left a comment.

It's a bit hard to test this, since on the OSS the licensing info is not available. It can be tested against this Enterprise branch: https://github.com/grafana/grafana-enterprise/pull/1878

File: public/app/features/admin/UserListAdminPage.tsx (hunk 1 above)
Comment (Clarity-89) [reply public/app/features/admin/UserListAdminPage.tsx:184]: I guess they'd be abstracted into a function.

Comment (hugohaggmark) [reply public/app/features/admin/UserListAdminPage.tsx:184]: That could be a good start 👍

**Comment by hugohaggmark** [comment]
Now that I've tested the PR, I've noticed that the `Belongs to` column has very little useful information for someone with a vision impairment. Also, the information varies because it can have links to edit the user or not, depending on the content and when it only holds icons and number of orgs/teams there is no tab stop. 

Not sure what the best solution is for this or if we need to solve this in this PR, but it would be great if we could announce how many orgs/teams the current `Belongs to` cell contains.

**Review by hugohaggmark — APPROVED** [review]
Works as expected when I test it.

**Comment by Clarity-89** [comment]
> Not sure what the best solution is for this or if we need to solve this in this PR, but it would be great if we could announce how many orgs/teams the current `Belongs to` cell contains.

Good point, we can add some accessible data about unit memberships.

**Comment by Clarity-89** [comment]
> We need updated doc content and an image.

This is an Enterprise feature, so will be added to Enterprise docs.

**Pull request merged**